docs/user/kafka/confluent.rst

```diff
@@ -163,3 +163,62 @@ Example GeoMesa Avro Schema
         }
       ]
     }
+
+Schema Overrides Config
+-----------------------
+
+The schema used to generate a ``SimpleFeatureType`` may optionally be overridden per topic by adding a data store
```

Comment:
Is the override the entire schema or does it modify things?  (The example makes it look like it is the entire schema.)

Reply:
It does override the whole thing, I could make it per field, that would just be a little more work. Should I?

Reply:
I think this is fine as is.

Reply:
As long as the docs are clear about how it all works.